tests/python/unittest/test_executor.py

```diff
@@ -118,20 +112,20 @@ def check_bind(disable_bulk_exec):
     check_bind(False)
 
 
-@with_seed(0)
+@with_seed()
```

Comment:
one nit: please also add something similar to this: https://github.com/apache/incubator-mxnet/pull/11888/files#diff-cb652780258e73a9cd08568f38929aa2R2509 to keep track of this flaky test.